src/styles/styles.ts

```diff
@@ -1395,7 +1395,7 @@ const styles = (theme: ThemeColors) =>
         createMenuPositionSidebar: (windowHeight: number) =>
             ({
                 horizontal: 18,
-                vertical: windowHeight - 75,
+                vertical: windowHeight - 87,
```

Comment:
Really don't like these random numbers being thrown around, any chance we can construct this value out of existing constants? Why is it changing from 75 => 87?

Reply:
It was mentioned in the description of this PR. But you're right it could be written more readably, so I refactored this code and added a comment with some explanations

Reply:
@jasperhuangg Could you check if the last change in this line is ok?